### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by gonzalojaubert**
Update Booster dependencies:

* jsonwebtoken
* underscore
* mocha
* jwks-rsa

### CODE REVIEW

File: common/changes/@boostercloud/framework-core/update_dependencies_2023-07-24-09-16.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@boostercloud/framework-core",
+      "comment": "Update booster dependencies",
+      "type": "minor"
```
Comment (javiertoledo) [inline common/changes/@boostercloud/framework-core/update_dependencies_2023-07-24-09-16.json:6]: Do we need to perform a release that is just upgrading dependencies? Unless these dependencies were upgraded for security reasons, I guess we can set this to "none" and get these upgraded dependencies on the next release along with some features 🤔

**Review by javiertoledo — APPROVED** [review]
LGTM, let's merge this ASAP as it doesn't change any behavior and all tests are passing. The only detail is that I'm not sure it's worth creating a minor release with this, but I'll let you decide @gonzalogarciajaubert.

**Review by charlietfe — APPROVED** [review]

**Review by alvaroloes — APPROVED** [review]

File: common/changes/@boostercloud/framework-core/update_dependencies_2023-07-24-09-16.json (hunk 1 above)
Comment (gonzalogarciajaubert) [reply common/changes/@boostercloud/framework-core/update_dependencies_2023-07-24-09-16.json:6]: It's a security update

**Pull request merged**